src/core/operations/HeatmapChart.mjs

```diff
@@ -0,0 +1,263 @@
+/**
+ * @author tlwr [toby@toby.codes]
+ * @copyright Crown Copyright 2019
+ * @license Apache-2.0
+ */
+
+import * as d3 from "d3";
```

Comment:
Would we get a benefit from only importing specfic symbols here? @n1474335 added a bundle size analyser that I think we can use to check